### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by 0ArturkA** [excaliburjs/example-ts-webpack#4]
Where need save textures/images? When i try add texture `new ex.Texture('./textures/texture.png')`, i got error `[Error] :  Failed to load resource  ./textures/texture.png  server responded with error code 404`

**Comment by eonarheim** [comment]
@0ArturkA Excalibur will look relative to where the main script is running. If the `./textures/texture.png` file is being served and the main script is at `./main.js` it should find and load it.

It's possible we'll need to update the webpack config to include additional files so the HTML plugin can serve them.

**Comment by 0ArturkA** [comment]
@eonarheim From what else can it be?

**Comment by eonarheim** [comment]
I'll have to take a look later, but I think that the webpack dev server
isn't serving the static images

**Comment by 0ArturkA** [comment]
@eonarheim Yes, webpack ignoring images. Please fix, this error greatly hampers the development.

**Comment by kamranayub** [comment]
This works but the idiomatic way to do this in webpack is the file loader.
This will import files, copy to output, and return the public path you can
use to pass to Excalibur resources.

https://github.com/webpack-contrib/file-loader/blob/master/README.md

**Comment by eonarheim** [comment]
I'll update the example today

**Comment by 0ArturkA** [comment]
@eonarheim Can you add file-loader to example?

**Comment by eonarheim** [comment]
@0ArturkA Apologies, life became busier than expected, I've updated master with a file loader example

**Comment by 0ArturkA** [comment]
@eonarheim Thank you.
